httplib.h

```diff
@@ -6647,6 +6690,49 @@ inline bool ClientImpl::process_request(Stream &strm, Request &req,
   return true;
 }
 
+inline ContentProviderWithoutLength ClientImpl::get_multipart_content_provider(
```

Comment:
This seems like a lot to be inlining but for some reason I get a linker multiple definition error if I don't.

Reply:
Might have something to do with the way split.py works?

Reply:
This usage of 'inline' isn't for actual inlining, but avoiding the multiple definition error when more than one .cpp files include httplib.h. So no problem!